src/adagrams.js

```diff
@@ -1,15 +1,109 @@
-export const drawLetters = () => {
-  // Implement this method for wave 1
+const LETTER_POOL = {
+  A: 9,  B: 2,  C: 2,  D: 4,  E: 12,
+  F: 2,  G: 3,  H: 2,  I: 9,  J: 1,
+  K: 1,  L: 4,  M: 2,  N: 6,  O: 8,
+  P: 2,  Q: 1,  R: 6,  S: 4,  T: 6,
+  U: 4,  V: 2,  W: 2,  X: 1,  Y: 2,
+  Z: 1,
+};
+
+const points = {
```

Comment:
For the `points` object:
- To clean up this file you could move this variable (and `LETTER_POOL`) into its own. After that you could import into the file, the type of import you will use will be dependent on the file type and export you use (if JS file). 

- In Javascript, we have the `const` keyword, meaning its value is a fixed piece of data (cannot be reassigned). It's great that you used it here. Since this variable is a true constant, we can take it a step further with the naming convention, like `LETTER_POOL`. By naming it using [convention](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const#basic_const_usage) of all uppercase and underscores, you let other developers know that this variable shouldn't be re-assigned or modified.
```suggestion
const POINTS = {
```